### PR TITLE
Add gov-contracts

### LINKED ISSUE #45: Prediction algorithms can return negative stock prices
Issue outlined in the title, may be worth setting a bottom bound on the prices to default to 0 if prediction is <0 

image

### PR DESCRIPTION
**Pull request by DidierRLopes**
This PR includes:
* Fix #45 for all prediction techniques
* Implement contracts commands

### CODE REVIEW

File: gamestonk_terminal/prediction_techniques/knn_view.py [hunk 1]
```diff
@@ -151,7 +151,9 @@ def k_nearest_neighbors(other_args: List[str], s_ticker: str, df_stock: pd.DataF
         preds = knn.predict(X_valid.reshape(X_valid.shape[0], X_valid.shape[1]))
         forecast_data = knn.predict(forecast_data_input.reshape(1, -1))
 
-        forecast_data_df = pd.DataFrame(forecast_data.T, index=future_dates)
+        forecast_data_df = pd.DataFrame(
+            [(i > 0) * i for i in forecast_data.T], index=future_dates
+        )
```
Comment (jmaslek) [inline gamestonk_terminal/prediction_techniques/knn_view.py:156]: I would do max(0, i) instead

Comment (DidierRLopes) [reply gamestonk_terminal/prediction_techniques/knn_view.py:156]: This one is more optimised

Captura de ecrã 2021-05-20, às 23 28 06

Comment (jmaslek) [reply gamestonk_terminal/prediction_techniques/knn_view.py:156]: Can be faster ;)

`[i if i>0 else 0 for i in ...]`

Screen Shot 2021-05-20 at 6 59 57 PM

Comment (DidierRLopes) [reply gamestonk_terminal/prediction_techniques/knn_view.py:156]: Then, I agree :D

**Review by aia — APPROVED** [review]

**Comment by jmaslek** [comment]
So it may just be my font, but the output is kinda weird
Screen Shot 2021-05-21 at 10 57 35 AM

**Comment by DidierRLopes** [comment]
> So it may just be my font, but the output is kinda weird
> Screen Shot 2021-05-21 at 10 57 35 AM

Not sure what to do of this?

Should we still merge it? Or, do you have any suggestion?

**Comment by jmaslek** [comment]
I'm good with merging.

**Pull request merged**